source/sfp/4_vegetation/sfp_408_nl_lsvi_stilwater/07_werkwijze.Rmd

```diff
@@ -0,0 +1,236 @@
+
+Indien verschillende habitattypen en/of subtypen voorkomen, wordt voor elk van de typen de verstoringspercentages op plasniveau ingeschat.
+Voor bijvoorbeeld de gezamenlijke aanwezigheid van 3130_na als 3130_aom in een plas, worden eutrofiëringspercentages voor beide subtypes op de volledige plas ingeschat.
+
+Voor habitattype 2190_a wordt het percentage van verschillende **vegetatielagen** ingeschat:
+
+-   (bodemwortelende) **Submerse vegetatie**: procentuele oppervlakte van de plas bedekt met ondergedoken en wortelende vegetatie; dit is inclusief *Ceratophyllum* en exclusief *Utricularia*.
+
+-   **Kranswieren**: oppervlakte van de plas bedekt met Characeae.
+    Opgelet, deze zijn ook vervat in het percentage submerse vegetatie, dus het percentage submerse vegetatie is steeds groter dan of gelijk aan het percentage kranswieren.
+
+-   **Overhangende bomen en struiken**: dit is de som van de oppervlakten van het wateroppervlak en de langdurig geïnundeerde oever dat beschaduwd wordt door overhangende bomen/struiken, dus deze die zich niet in de plas bevinden.
+    Bij het bepalen van de beschaduwing wordt een verticale projectie van de overhangende boom/struiken gebruikt.
+    Dit geeft een indicatie van slechte lichtomstandigheden in het water, en daardoor vegetatiegroei belemmert.
+
+-   **Bomen en struiken**: percentage bedekking van het wateroppervlak en de langdurig geïnundeerde oever met bomen of struiken; enkel deze die zich in de plas bevinden of die in de geïnundeerde oever wortelen.
+    Dit is een indicatie van successie.
+
+### Structuurkenmerken
+
+#### Vertical structuur
+
+De verticale structuur brengt de successie in rekening.
+Een overmatige aanwezigheid van hoger groeiende, meer concurrentiekrachtige planten bemoeilijkt de groei van de kenmerkende isoëtiden en dwergbiezen:
+
+-   **Isoëtiden**: Voor HT3110 wordt de bedekking van isoëtiden versus andere groeivormen genoteerd, oftewel door een Tansley-inschatting per groep te noteren of door aan te kruisen of het aandeel isoëtiden groter of kleiner is dan de andere groeivormen.
+
+-   **‘Opgaande’ soorten**: voor habitattype 3130_na wordt de overgroeiing van de vegetatievlekken (definitie: zie verder) door hoger groeiende niet-sleutelsoorten ingeschat.
+    Dit is de vegetatie hoger dan 15 cm en kan naast helofyten ook bestaan uit submerse vegetatie die zorgt voor concurrentie.
+
+Een overdadige ontwikkeling van **helofyten** zorgt voor concurrentie voor licht en ruimte, waardoor de groei van ondergedoken en drijvende vegetatie wordt bemoeilijkt.
+
+De bedekking van helofyten (exclusief sleutelsoorten) wordt als percentage ingeschat bij habitattype 3130_aom, 3140 en 3150; hierbij wordt dit percentage ingeschat op plasniveau; m.n.
+% bedekking ten opzichte van het wateroppervlak.
+Ook bij habitattype 2190_a komt dit aan bod als de bedekking van **robuuste monocotylen**.
+
+#### Horizontale structuur
```

Comment:
zie hoger: Hier kan de kop (#####) misschien best gewist worden; dus in het vet zetten en de tekst op een nieuwe regel beginnen. Anders gaan de koppen misschien teveel cijfers bevatten?